==== triton/test/Conversion/tritongpu_to_llvm.mlir#19 - /google/src/cloud/ecg/mlir_2aec7083ada09c8b8a0aad79492cbedcf8f9fbb7_1704832781/triton/test/Conversion/tritongpu_to_llvm.mlir ====
# action=edit type=text
--- triton/test/Conversion/tritongpu_to_llvm.mlir	2023-12-15 04:43:34.000000000 -0800
+++ triton/test/Conversion/tritongpu_to_llvm.mlir	2024-01-09 12:40:14.000000000 -0800
@@ -3,7 +3,7 @@
 module attributes {"triton_gpu.num-ctas" = 1 : i32, "triton_gpu.num-warps" = 4 : i32} {
   // CHECK: llvm.func @test_empty_kernel(%arg0: i64, %arg1: !llvm.ptr<1>)
   // Here the 128 comes from the 4 in module attribute multiples 32
-  // CHECK: nvvm.kernel = 1 : ui1, nvvm.maxntid = [128 : i32]
+  // CHECK: nvvm.kernel = 1 : ui1, nvvm.maxntid = array<i32: 128>
   tt.func @test_empty_kernel(%lb : index, %A : !tt.ptr<f16>) {
     // CHECK:  llvm.return
     tt.return
